src/wallet/components/Wallet.tsx

```diff
@@ -43,6 +43,17 @@ const WalletContent = ({ children }: WalletReact) => {
 };
 
 export const Wallet = ({ children }: WalletReact) => {
+  const [isClient, setIsClient] = useState(false);
+
+  useEffect(() => {
+    setIsClient(true);
```

Comment:
Let's try to solve this for 2~3 components at the same time.

As I see this being an issue for Identity components as well.

Reply:
i tested `Identity` in my app and it didn't seem to be an issue